.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,33 @@
+name: Pull Request
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  conventional-title:
+    name: Validate PR title is Conventional Commit
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check title
+        if: github.event_name == 'pull_request_target'
+        uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            fix
+            feat
+            chore
+            test
+            perf
+            refactor
+            docs
+            ci
```

Comment:
Some of these usually fall under scopes and not types -- ie chore(test) and chore(ci)

Reply:
Yeah I think I like these types despite them sometimes falling under scope. `ci` however I think could not be a type / stay as a scope. I'm currently unsure whether or not we should require scopes, e.g.:
```yml
          # Configure that a scope must always be provided.
          requireScope: true
```
But I'm leaning towards it, since it would help when reading commit history, creating changelogs and release notes etc.

Reply:
I have no strong opinion on requireScope -- Yep release notes (and optionally releases) become essentially automatic with scopes and types using release-plz or release-please